File: java/basic_app/app/src/main/java/com/kuku/muku/kukumukubasicapp/PeachesActivity.java
package com.kuku.muku.kukumukubasicapp;

import android.os.Bundle;

import com.kuku.muku.kukumukubasicapp.R;

public class PeachesActivity extends FruitActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setStaticAttributes("peaches");
    }

    @Override
    protected int getLayoutResourceId() {
        return R.layout.activity_peaches;
    }
}

File: kotlin/basic_app/app/src/main/java/com/example/onelinkbasicapp/PeachesActivity.kt
package com.example.onelinkbasicapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class PeachesActivity : FruitActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }
    //Returns the id of this class so the fruit activity can change how it look
    override fun getLayoutResourceId(): Int {
        return R.layout.activity_peaches
    }
}